components/SideBar.tsx

```diff
@@ -97,6 +99,18 @@ const Sidebar = () => {
               svg={<EditIcon />}
               currentPath={router.asPath}
             />
+            <MenuLink
+              linkProps={{ href: '/sites' }}
+              label="My Sites"
+              svg={<SiteListIcon color={'white'} />}
+              currentPath={router.asPath}
+            />
+            <MenuLink
+              linkProps={{ href: '/more-info' }}
+              label="More Info"
+              svg={<SearchIcon color={'white'} />}
+              currentPath={router.asPath}
+            />
```

Comment:
could we actually add these two new links in the same section as the "dashboard" link? I think they'd make more sense up there, as they're linking to other pages (like the dashboard) rather than actions that users can take.